api/specs/webserver/v0/components/schemas/datasets.yaml

```diff
@@ -0,0 +1,26 @@
+DatasetMetaDataEnveloped:
+  type: object
+  required:
```

Comment:
Notice that you use required fields differently for your "enveloped" schemes ... For the moment is not critical but the day we reactivate validation in storage, it might cause problems. Just keep it in mind